app/mailers/feedback_mailer.rb

```diff
@@ -2,8 +2,9 @@ class FeedbackMailer < ApplicationMailer
   def send_message
     @name = params[:name]
     @email = params[:email]
+    @subject = params[:subject]
     @message = params[:message]
 
-    mail to: "devs@email.com", subject: "New Support Message"
+    mail to: "devs@email.com", subject: @subject
```

Comment:
This reminds me, we should set up ENV variables in another issue. Then we can add the homewardtails email address here.